diff --git a/node_modules/postcss/lib/input.js b/node_modules/postcss/lib/input.js
index 7e59a2bc8e16..5d2abe01bc26 100644
--- a/node_modules/postcss/lib/input.js
+++ b/node_modules/postcss/lib/input.js
@@ -1,6 +1,6 @@
 'use strict'
 
-let { nanoid } = require('nanoid/non-secure')
+let { nanoid } = require('nanoid/non-secure/index.cjs')
 
 let terminalHighlight = require('./terminal-highlight')
 let CssSyntaxError = require('./css-syntax-error')
